modules/doctest/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://bit.ly/doctest-docs",
```

Comment:
This looks medium fishy, could you use the GitHub repo instead?

Reply:
I replaced it with https://github.com/doctest/doctest

The old URL was mentioned here:

![image](https://github.com/bazelbuild/bazel-central-registry/assets/3775001/81bb00e7-2cbf-4ea9-809e-72bffe81182a)